idx/memory/memory.go

```diff
@@ -1092,7 +1179,7 @@ func (m *UnpartitionedMemoryIdx) Find(orgId uint32, pattern string, from int64)
 	var matchedNodes []*Node
 	var err error
 	m.RLock()
-	defer m.RUnlock()
+	m.RUnlock()
```

Comment:
I think that needs to be moved to below 1183

Reply:
actually, this should just still have the defer. I added i back